test_conformance/math_brute_force/common.cpp

```diff
@@ -67,22 +67,28 @@ void EmitDefineUndef(std::ostringstream &kernel, const char *name,
     kernel << "#define " << name << " " << GetUndefValue(type) << '\n';
 }
 
-void EmitEnableExtension(std::ostringstream &kernel, ParameterType type)
+void EmitEnableExtension(std::ostringstream &kernel,
+                         const std::initializer_list<ParameterType> &types)
 {
-    switch (type)
-    {
-        case ParameterType::Double:
-            kernel << "#pragma OPENCL EXTENSION cl_khr_fp64 : enable\n";
-            break;
+    bool needsFp64 = false;
 
-        case ParameterType::Float:
-        case ParameterType::Int:
-        case ParameterType::UInt:
-        case ParameterType::Long:
-        case ParameterType::ULong:
-            // No extension required.
-            break;
+    for (const auto &type : types)
+    {
+        switch (type)
+        {
+            case ParameterType::Double: needsFp64 = true; break;
+
+            case ParameterType::Float:
+            case ParameterType::Int:
+            case ParameterType::UInt:
+            case ParameterType::Long:
+            case ParameterType::ULong:
+                // No extension required.
```

Comment:
I've noticed this test does not verify neither `gHasLong` nor `cles_khr_int64` extension ? Is it unnecessary in here ?